### ISSUE TITLE
Provide option to avoid automatically added source package reviews when creating a submit request

### ISSUE THREAD

**Issue by jberry-suse** [openSUSE/open-build-service#8215]
Currently, OBS will automatically add a review for the source package when a submit request is created if the source project does not contain the `OBS:ApprovedRequestSource` attribute. Given all the tooling openSUSE/SLE has for adding the appropriate reviewer this functionality has likely outlived its purpose. The policy should be up to the target project anyway. If I submit something from a devel project to my home project there is no need for the devel project to review it. In general this review is never desired.

For Factory, `factory-auto` will a review for the maintainer if not submitted by one. For leap it is only added if not already coming from a product (Factory/SLE) that would have already required that review.

My preference would be the review is simply never added as I imagine the original reason was to aid in Factory devel process which is no longer needed. Alternatively, an API option to skip the addition of such reviews would be acceptable.

During the later staging of Leap, it is desired to be able to submit maintenance updates against the previous Leap version before they are released to speed them through the staging process. All the tooling in place to find and submit such updates is available from origin-manager, but needless reviews are added for the maintenance incident project (or devel project if submitting pending Factory requests).

Given I can do the following to get around this it cannot be argued as some sort of protection measure.

```
osc copypack source_project source_package my_home_project
osc sr my_home_project source_package target_project
```

This gets around it since submitted by a maintainer of the package.

Without this feature/change we are blocked in utilizing this origin-manager feature. Related tracking issue: openSUSE/openSUSE-release-tools#2146.

**Comment by coolo** [comment]
I agree, but knowing how concerned @adrianschroeter is about compatiblity - can we have an attribute in Factory/Leap OBS:ApprovedRequestTarget and deprecate this review adding?
